frontend/src/layout/navigation/TopBar/SitePopover.tsx

```diff
@@ -55,14 +58,17 @@ function AccountInfo(): JSX.Element {
                     {user?.email}
                 </div>
             </div>
-            <Link
-                to={urls.mySettings()}
-                onClick={closeSitePopover}
-                className="SitePopover__side-link"
-                data-attr="top-menu-item-me"
-            >
-                Manage account
-            </Link>
+            <Tooltip title="Account settings">
+                <Link
```

Comment:
This should be a `LemonButton` for better interactivity. It also supports the `to` prop, so there's no need for wrapping it with a link. The icon should be in the `icon` prop.